src/docs/codelabs/layout-basics.md

```diff
@@ -1,427 +1,646 @@
+  but at different font sizes and in different colors.
+  `Row` specifies the `crossAxisAlignment` and `textBaseline` properties. 
 
-<iframe src="https://dartpad.dev/experimental/embed-new-flutter.html?id=326b8c5774079b7a80922e11a3730f99" width="100%" height="400px"></iframe>
+  **1.** Click the **Run** button. 
 
-### Spacers expand to make space
+  **2.** Change `CrossAxisAlignment.center` to `CrossAxisAlignment.baseline`, and run again.
+{{site.alert.end}}
+{% comment %}
+  Gist: https://gist.github.com/datafoya/0ff109090b99ef1873d9fad501b2bc86
+{% endcomment %}
+<iframe src="https://dartpad.dev/experimental/embed-new-flutter.html?id=0ff109090b99ef1873d9fad501b2bc86&amp;theme=dark&amp;split=60" width="100%" height="400px"></iframe>
+
+## Icon widget
+
+The `Icon` widget displays a graphical symbol 
+that represents an aspect of the UI. 
+Flutter is preloaded with icon packages for 
+[Material](https://api.flutter.dev/flutter/material/MaterialApp-class.html) and 
+[Cupertino](https://api.flutter.dev/flutter/cupertino/CupertinoApp-class.html) applications.
+
+#### Example: Creating an Icon
+{:.no_toc}
+{{site.alert.secondary}}
+  The following example displays the widget `Icons.widget` from the
+  [Material Icon library](https://api.flutter.dev/flutter/material/Icons-class.html) in red and blue.
+
+  **1.** Click the **Run** button. 
+
+  **2.** Add another `Icon` from the 
+  [Material Icon library](https://api.flutter.dev/flutter/material/Icons-class.html)
+  with a size of 50. 
+
+  **3.** Give the `Icon` a color of `Color.amber` from the 
+  [Material Color palette](https://api.flutter.dev/flutter/material/Colors-class.html), 
+  and run again.  
+{{site.alert.end}}
+{% comment %}
+  Gist: https://gist.github.com/datafoya/01688fca8c13f85d93078054af2e858b
+{% endcomment %}
+<iframe src="https://dartpad.dev/experimental/embed-new-flutter.html?id=01688fca8c13f85d93078054af2e858b&amp;theme=dark&amp;split=60" width="100%" height="400px"></iframe>
 
-`Spacers` are another convenient way to make space between
-items in a `Row`.  They're flexible, and expand to fill any
-leftover space.
+## Image widget 
 
-#### Code example
+The `Image` widget displays an image. You either can reference images using a URL,
+or you can include images inside your app package. Since DartPad can't package an image,
+the following example uses an image from the network.
 
-Try adding a `Spacer` in between the first and second children of the
-`Row` below.
+#### Example: Displaying an image
+{:.no_toc}
+{{site.alert.secondary}}
+  The following example displays an image that's stored remotely on [GitHub](https://github.com/flutter/website/tree/master/examples/layout/sizing/images). 
+  The `Image.network` method takes a string parameter that contains the image's URL.
 
-<iframe src="https://dartpad.dev/experimental/embed-new-flutter.html?id=dd68c1eb491e7a22a2ceb4127d78e504" width="100%" height="400px"></iframe>
+  In this example, `Image.network` contains a short Url. 
```

Comment:
Url -> URL (global, unless you're specifically referring to an instance of a class named `Url`, in which case it should perhaps be code font.